packages/presentation/lib/src/star_movie_app.dart

```diff
@@ -44,22 +48,45 @@ class _StarMovieAppState extends BlocScreenState<StarMovieApp, AppBloc> {
               child: CircularProgressIndicator(),
             );
           return Scaffold(
-            bottomNavigationBar: tile.isShowNavBar
-                ? AppNavigationBar(
-                    bloc: bloc,
-                    bottomNavIndex: tile.bottomNavIndex,
-                  )
-                : null,
-            body: Navigator(
-              onPopPage: (route, result) {
-                bloc.handleRemoveRouteSettings(route.settings);
-                return route.didPop(result);
-              },
-              pages: tile.pages.toList(),
+            key: _scaffoldKey,
+            bottomNavigationBar: bottomNavBarHandler(context, tile),
+            body: Row(
+              children: [
+                sideNavBarHandler(context, tile),
+                Expanded(
+                  child: Navigator(
+                    onPopPage: (route, result) {
+                      bloc.handleRemoveRouteSettings(route.settings);
+                      return route.didPop(result);
+                    },
+                    pages: tile.pages.toList(),
+                  ),
+                ),
+              ],
             ),
           );
         },
       ),
     );
   }
+
+  Widget? bottomNavBarHandler(BuildContext context, AppData tile) {
+    return WidgetDisplayHelper.isBottomNavBarActive(context) &&
+            tile.isShowNavBar
+        ? AppNavigationBar(
+            bloc: bloc,
+            bottomNavIndex: tile.navIndex,
+          )
+        : null;
+  }
+
+  Widget sideNavBarHandler(BuildContext context, AppData tile) {
+    return !WidgetDisplayHelper.isBottomNavBarActive(context) &&
+            tile.isShowNavBar
+        ? SideNavigationBar(
+            bloc: bloc,
+            selectedIndex: tile.navIndex,
+          )
+        : SizedBox.shrink();
+  }
```

Comment:
better create class https://stackoverflow.com/questions/53234825/what-is-the-difference-between-functions-and-classes-to-create-reusable-widgets and https://www.youtube.com/watch?v=IOyq-eTRhvo&ab_channel=Flutter